src/main/java/org/takes/http/MainRemote.java

```
@@ -83,7 +83,9 @@ public MainRemote(final Class<?> type, final String... passed) {
      */
     public void exec(final MainRemote.Script script) throws Exception {
         final File file = File.createTempFile("takes-", ".txt");
-        file.delete();
+        if (!file.delete()) {
+            file.deleteOnExit();
```

@yegor256 ok let's throw an exception then